src/hotspot/share/prims/whitebox.cpp

```diff
@@ -2520,6 +2521,22 @@ WB_ENTRY(void, WB_UnlockCritical(JNIEnv* env, jobject wb))
   GCLocker::unlock_critical(thread);
 WB_END
 
+WB_ENTRY(jboolean, WB_SetVirtualThreadsNotifyJvmtiMode(JNIEnv* env, jobject wb, jboolean enable))
+  if (!Continuations::enabled()) {
+    tty->print_cr("WB error: must be Continuations::enabled()!");
+    return JNI_FALSE;
+  }
+  jboolean result = false;
+#if INCLUDE_JVMTI
+  if (enable) {
+    result = JvmtiEnvBase::enable_virtual_threads_notify_jvmti();
+  } else {
+    result = JvmtiEnvBase::disable_virtual_threads_notify_jvmti();
+  }
+#endif
+  return result;
```

Comment:
The test never check results, so it might be better to generate fatal error and fail here/throw exception to catch bug earlier?

Reply:
Thank you for the comment.
I think, the decision to throw exception has to be in the test.
Modified the test as below:
```
<<<        WB.setVirtualThreadsNotifyJvmtiMode(enable);
 ---
>>>        boolean status = WB.setVirtualThreadsNotifyJvmtiMode(enable);
>>>        if (!status) {
>>>            throw new RuntimeException("Java: failed to set VirtualThreadsNotifyJvmtiMode: " + enable);
>>>        }
```